persons/record.html

```diff
@@ -1,29 +1,89 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
-    <div data-template="app:fix-links" class="record">
-        <div class="interior-content">
-            <div data-template="app:other-data-formats" data-template-formats="print,tei,rdf,text"/>
-            <div data-template="app:h1"/>
-            <div data-template="app:data-visualization" data-template-mode="force" data-template-locus="single"/>
-            <div data-template="app:display-nodes" data-template-paths="descendant::tei:text"/>
-            <!--
+    <div class="grey-dark">
```

Comment:
The persons record should revert to the version I had up originally, not the version from the old app. We do not use there is no longer a separate xquery module for persons, it was unnecessary. 

This is the essential code: 
```
<div data-template="app:other-data-formats" data-template-formats="print,tei,rdf,text"/>
            <div data-template="app:h1"/>
            <div data-template="app:data-visualization" data-template-mode="force" data-template-locus="single"/>
            <div data-template="app:display-nodes" data-template-paths="descendant::tei:text"/>
```

Reply:
See my comments.

Reply:
So should I just discard the file from the old app? Or only revert the changes in it that you marked?

Reply:
Discard the file from the old app.